widgetry/src/backend_glow_native.rs

```diff
@@ -12,16 +12,22 @@ pub fn setup(window_title: &str) -> (PrerenderInnards, winit::event_loop::EventL
     // TODO If people are hitting problems with context not matching what their GPU provides, dig up
     // backend_glium.rs from git and bring the fallback behavior here. (Ideally, there'd be
     // something in glutin to directly express this.) multisampling: 2 looks bad, 4 looks fine
-    let context = match glutin::ContextBuilder::new()
+    let context = glutin::ContextBuilder::new()
         .with_multisampling(4)
         .with_depth_buffer(2)
-        .build_windowed(window, &event_loop)
-    {
-        Ok(ctx) => ctx,
-        Err(err) => {
-            panic!("Your videocard doesn't support the OpenGL mode requested. This is a common issue when running inside a virtual machine; please run natively if possible. See https://github.com/dabreegster/abstreet/issues/103 for more info, and feel free to ask for help using that issue.\n\nError: {}", err);
-        }
-    };
+        .build_windowed(window.clone(), &event_loop).map_err(|err| {
+            warn!("Trying default graphics context after standard graphics context failed with error: {:?}",  err);
+            glutin::ContextBuilder::new().build_windowed(window.clone(), &event_loop)
+    })
+        .map_err(|err| {
```

Comment:
Does glutin provide an error type that we can deconstruct?

Reply:
Indeed - looks like this method returns a CreationError: https://docs.rs/glutin/0.25.1/glutin/enum.CreationError.html

At the moment we're not doing anything smart with it, as a first step we're just logging it and trying something else to hopefully help in future debugging cases. 

If this approach doesn't work for people, we could spend more time trying to be smart and handling the particular cases.